src/logic/operator.js

```diff
@@ -0,0 +1,22 @@
+const Big = require('big.js')
+
+const operator = (numberOne, numberTwo, operation) => {
+  const bigNumberOne = Big(numberOne)
+  const bigNumberTwo = Big(numberTwo)
+  switch (operation) {
+    case '+':
+      return bigNumberOne.plus(bigNumberTwo).toString()
+    case '-':
+      return bigNumberOne.minus(bigNumberTwo).toString()
+    case '/':
```

Comment:
If `numberOne` is zero and `numberTwo` is zero this will throw a `DividedByZero` error.
Kindly handle this error